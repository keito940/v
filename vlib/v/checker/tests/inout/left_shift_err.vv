fn main() {
	l := []int
	l << 'test'
}
