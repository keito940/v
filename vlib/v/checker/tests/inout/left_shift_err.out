vlib/v/checker/tests/inout/left_shift_err.v:3:7: error: cannot shift type string into array_int
    1| fn main() {
    2|     l := []int
    3|     l << 'test'
                ~~~~~~
    4| }